packages/bruno-app/src/components/BrunoSupport/index.js

```diff
@@ -29,7 +29,7 @@ const BrunoSupport = ({ onClose }) => {
           <div className="mt-2">
             <a href="https://github.com/usebruno/bruno" target="_blank" className="flex items-end">
               <IconBrandGithub size={18} strokeWidth={2} />
-              <span className="label ml-2">Github</span>
+              <span className="label ml-2">GitHub</span>
```

Comment:
brand -- note that I'm not changing things like `IconBrandGithub` that requires too much thought.